data-prepper-core/src/main/java/com/amazon/dataprepper/parser/model/DataPrepperConfiguration.java

```diff
@@ -47,7 +49,8 @@ public DataPrepperConfiguration(
             @JsonProperty("serverPort") final String serverPort,
             @JsonProperty("metricRegistries") final List<MetricRegistryType> metricRegistries,
             @JsonProperty("authentication") final PluginModel authentication,
-            @JsonProperty("metricTags") final Map<String, String> metricTags
+            @JsonProperty("metricTags") final Map<String, String> metricTags,
+            @JsonProperty("peer_forwarder") final PeerForwarderConfiguration peerForwarderConfiguration
```

Comment:
If core peer-forwarder support is starting from 2.0 I am OK with snake_eyes here.

Reply:
Yeah, we are supporting it in 2.0